src/components/KToaster/ToastManager.ts

```diff
@@ -34,15 +34,15 @@ export default class ToastManager {
     this.toaster = createVNode(KToaster, {
       toasterState: this.toasts.value,
       zIndex: options?.zIndex ? options.zIndex : defaultZIndex,
-      onClose: (key: any) => this.close(key),
+      onClose: (key: string) => this.close(key),
     })
 
     if (this.toastersContainer) {
       render(this.toaster, this.toastersContainer)
     }
   }
 
-  setTimer(key: any, timeout: number): number {
+  setTimer(key: string, timeout: number): number {
```

Comment:
question: the interface is changing here; is that going to create a downstream issue?